pkg/ocihook/ocihook.go

```diff
@@ -365,6 +366,16 @@ func getMACAddressOpts(opts *handlerOpts) ([]gocni.NamespaceOpts, error) {
 func onCreateRuntime(opts *handlerOpts) error {
 	loadAppArmor()
 
+	if opts.state.Annotations[labels.Name] != "" {
```

Comment:
Q: Is it possible to have labels.Name as an empty string; should we throw an error if this annotation is empty string?

Reply:
https://github.com/containerd/nerdctl/blob/1642bef99f0931b79fe75fd54c1566e9e061b178/pkg/labels/labels.go#L23-L30

It doesn't seem possible for it to be an empty string.

Reply:
it cannot be empty. So removed to avoid misunderstanding